fpdf/fpdf.py

```diff
@@ -47,10 +48,11 @@
 )
 
 LOGGER = logging.getLogger(__name__)
+HERE = Path(__file__).resolve().parent
```

Comment:
`PARENT_DIR` could be more explicit than just `HERE`

Reply:
It is `PARENT` or `DIR`, but `PARENT_DIR` sounds double level to me. I personally use and prefer the `HERE` name.